snuba/manual_jobs/toy_job.py

```diff
@@ -1,13 +1,14 @@
-from typing import Any
-
-import click
-
-from snuba.manual_jobs import Job
+from snuba.cli import logger
```

Comment:
nit: Not a big deal at this point but do you think we should have a separate logger for `manual_jobs`? The [second answer](https://stackoverflow.com/questions/15727420/using-logging-in-multiple-modules) to this question might be something we want to have, or we can follow the pattern from other modules.